Submitty/Submitty
# [Feature:Submission] Refactor Submission Page and Add Peer Grading Output

### This PR: 
1. Adds appropriate display of peer grading results to the student homework view/submission page.
2. Refactors the homework submission page to be more modular (screenshot below).
    1. Adds separate "islands" for Autograding, TA, and Peer Results.
    2. Adds an overview section.

### To Test:
1.  View results of peer, ta, and autograding and their permutations (ta but no autograding, autograding but no ta grading, etc.)
2. Check hidden testcase output before and after grades have been released.
3. Check version conflict output.
4. Check confetti. 

![Screenshot from 2020-02-27 13-20-44](https://user-images.githubusercontent.com/11758882/75506800-239e1b80-59ad-11ea-85ad-ce01bd202153.png)


closes #5108
